### ISSUE TITLE
valec tries to capture options after exec

### ISSUE THREAD

**Issue by potsbo** [dtan4/valec#49]
## WHY

`envchain wtd valec exec sample ls -l` fails because valec seems to be trying to interpret option `-l`
Because of this, I can not use `valec` in production.

## WHAT

@dtan4 Is this what it is, or considered as a bug?
If you say this is the design, what do you think is the best practice to use option after exec?

**Comment by dtan4** [comment]
Thank you for reporting. Maybe `-l` (and all of other flags...) are captured by cobra framework.
I'll try to disable capturing, or force to add hyphen before commands...

```
$ valec exec sample -- -ls -l
```

**Comment by dtan4** [comment]
Current Valec can execute commands with flag, using `--`.

```bash
$ valec exec sample -- ls -l
total 88
-rw-r--r--   1 dtan4  staff  3707  1 17 11:36 CHANGELOG.md
-rw-r--r--   1 dtan4  staff  1081 11 24 12:03 LICENSE
-rw-r--r--   1 dtan4  staff  1613  1 17 11:36 Makefile
-rw-r--r--   1 dtan4  staff  6110 12 22 20:44 README.md
-rw-r--r--   1 dtan4  staff   417 12  9 15:09 appveyor.yml
drwxr-xr-x   6 dtan4  staff   204 12 26 16:56 aws
drwxr-xr-x   3 dtan4  staff   102  1 17 15:17 bin
drwxr-xr-x  14 dtan4  staff   476  1 17 14:40 cmd
-rw-r--r--   1 dtan4  staff  7511 12 26 16:51 coverage.txt
drwxr-xr-x   9 dtan4  staff   306  1 17 15:19 dist
-rw-r--r--   1 dtan4  staff  2167 12 26 16:56 glide.lock
-rw-r--r--   1 dtan4  staff   452 12 26 16:56 glide.yaml
-rw-r--r--   1 dtan4  staff    82 11 24 14:18 main.go
drwxr-xr-x   4 dtan4  staff   136 12 26 19:21 msg
drwxr-xr-x   4 dtan4  staff   136 12 22 20:41 secret
drwxr-xr-x   7 dtan4  staff   238 12 22 20:41 testdata
drwxr-xr-x   5 dtan4  staff   170 12  2 15:46 tmp
drwxr-xr-x   4 dtan4  staff   136 12 26 19:21 util
drwxr-xr-x   5 dtan4  staff   170 12 26 14:42 vendor
drwxr-xr-x   4 dtan4  staff   136 12  9 15:09 version
```

**Comment by potsbo** [comment]
It does work.
Thank you for the information.

**Comment by dtan4** [comment]
According to https://github.com/spf13/cobra/issues/352 , `cmd.Flags().SetInterspersed(false)` can stop parsing flags after subcommand.

https://github.com/spf13/pflag/blob/51268031d79952516489a9f8aa34e9709b98d946/flag.go#L952-L955

> SetInterspersed sets whether to support interspersed option/non-option arguments.

Actually it works!

```
$ valec exec sample ls -l
total 88
-rw-r--r--   1 dtan4  staff  3707  1 17 11:36 CHANGELOG.md
-rw-r--r--   1 dtan4  staff  1081 11 24 12:03 LICENSE
-rw-r--r--   1 dtan4  staff  1613  1 17 11:36 Makefile
-rw-r--r--   1 dtan4  staff  6110 12 22 20:44 README.md
-rw-r--r--   1 dtan4  staff   417 12  9 15:09 appveyor.yml
drwxr-xr-x   6 dtan4  staff   204 12 26 16:56 aws
drwxr-xr-x   3 dtan4  staff   102  1 18 18:23 bin
drwxr-xr-x  14 dtan4  staff   476  1 18 18:23 cmd
-rw-r--r--   1 dtan4  staff  7511 12 26 16:51 coverage.txt
drwxr-xr-x   9 dtan4  staff   306  1 17 15:19 dist
-rw-r--r--   1 dtan4  staff  2167 12 26 16:56 glide.lock
-rw-r--r--   1 dtan4  staff   452 12 26 16:56 glide.yaml
-rw-r--r--   1 dtan4  staff    82 11 24 14:18 main.go
drwxr-xr-x   4 dtan4  staff   136 12 26 19:21 msg
drwxr-xr-x   4 dtan4  staff   136 12 22 20:41 secret
drwxr-xr-x   7 dtan4  staff   238 12 22 20:41 testdata
drwxr-xr-x   5 dtan4  staff   170 12  2 15:46 tmp
drwxr-xr-x   4 dtan4  staff   136 12 26 19:21 util
drwxr-xr-x   5 dtan4  staff   170 12 26 14:42 vendor
drwxr-xr-x   4 dtan4  staff   136 12  9 15:09 version

$ valec --table-name hogefuga exec sample ls -l
Failed to load secrets from DynamoDB. namespace=sample: Failed to list up secrets. namespace=sample: ResourceNotFoundException: Requestedresource not found
        status code: 400, request id: <hidden>
```

**Comment by dtan4** [comment]
Released as Valec v0.5.4.
